core/agent/src/main/java/org/apache/spark/raydp/Agent.java

```diff
@@ -72,8 +72,12 @@ public static void premain(String agentArgs, Instrumentation inst)
     String jobId = System.getenv("RAY_JOB_ID");
     String rayAddress = System.getProperty("ray.address");
     if (jobId != null && rayAddress != null) {
-      try (FileWriter writer = new FileWriter(logDir + "/java-worker-" + jobId + "-" + pid + ".log")) {
-        writer.write(":job_id:" + jobId + "\n");
+      String prefix = System.getProperty("ray.logging.file-prefix", "java-worker");
+      if ("java-worker".equals(prefix)) {
+        try (FileWriter writer = new FileWriter(logDir + "/" + prefix + "-" +
+                jobId + "-" + pid + ".log")) {
+          writer.write(":job_id:" + jobId + "\n");
```

Comment:
Can you please comment why we write the jobId here?

Reply:
done